src/main/java/nitezh/ministock/activities/ChartActivity.java

```diff
@@ -24,18 +38,92 @@
 public class ChartActivity extends Activity {
     // Public variables
     public static int mAppWidgetId = 0;
+    private final String alphavantagekey = "ZKD8M6L9CEQAK89H";
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         URL url = null;
         Bitmap bmImg = null;
```

Comment:
url and bmImg are not being used so should be removed

Reply:
fixed